builder/builder-next/adapters/containerimage/pull.go

```diff
@@ -82,10 +83,11 @@ func (is *Source) getResolver(hosts docker.RegistryHosts, ref string, sm *sessio
 	if res := is.resolverCache.Get(ref, g); res != nil {
 		return res
 	}
-	auth := resolver.NewSessionAuthenticator(sm, g)
-	r := resolver.New(hosts, auth)
-	r = is.resolverCache.Add(ref, auth, r, g)
-	return r
+	//auth := resolver.NewSessionAuthenticator(sm, g)
```

Comment:
Remove commented code

Reply:
Do you want me to squash https://github.com/moby/moby/commit/3756668adbbb48cf027a2229ba560d3777ed871e with vendor commit? I tried to make that commit understandable to see what it was doing before, while still keeping the vendor commit compilable.

Reply:
Oh I see. I didn't look at the final view, just per commit.